app/views/coronavirus_landing_page/components/shared/_section.html.erb

```diff
@@ -7,7 +7,7 @@
       margin_bottom: 4,
     } %>
   <% end %>
-  <ul class="covid__list covid__list--accordion">
+  <ul class="covid__list">
```

Comment:
Should we also remove the css that declared covid__list--accordion?

Reply:
Ah it doesn’t exist, I think it’s a stray tag.

Reply:
I see it in master: https://github.com/alphagov/collections/blob/ad6ca58abf14712d5ed623e04bea32a72fd998e1/app/assets/stylesheets/views/_covid.scss line 145

Reply:
:see_no_evil: I see it now - removed

Reply:
I think with the update you've lost the previous deletion from the file where .covid__accordion-link is declared.

Reply:
urgh - removed. Panic updating of course goes awry, on hindsight once re-opened I should have put into instantly into draft so I could wait for the release merge. #TIL

Reply:
No worries, glitches like this are what the process is here to spot. Like you say, let's wait for the release and double check visuals before approve/merge.